src/client/imagefilters.cpp

```diff
@@ -87,77 +94,73 @@ static void imageCleanTransparentWithInlining(video::IImage *src, u32 threshold)
 		}
 	};
 
-	Bitmap bitmap(dim.Width, dim.Height);
+	Bitmap seen(dim.Width, dim.Height);
+	Bitmap prev_levels(dim.Width, dim.Height);
+	std::vector<v2u32> level, next_level;
```

Comment:
thought: so the `Bitmap` class is essentially a `std::set<v2u32>`.
maybe it's faster to scan that for values than use a vector, because the bitmap is a fixed 32 bytes while the vector goes to 160 bytes if you store even 20 pairs in it